building/build-debs/homeworld-admin-tools/src/authority.py

```diff
@@ -23,7 +23,24 @@ def name_for_decrypted_file(name_of_encrypted_file):
     if name_of_encrypted_file.endswith(ENCRYPTED_EXTENSION):
         return name_of_encrypted_file[:-len(ENCRYPTED_EXTENSION)]
     raise ValueError("Filename " + name_of_encrypted_file + " does not have expected suffix '" + ENCRYPTED_EXTENSION + "'.")
-    
+   
+def validate_pem_file(dir_path: str, file_name: str) -> None:
+    full_file_name = os.path.join(dir_path, file_name)
```

Comment:
If all this needs is a filename, it should be passed in as a filename. There's no reason for this function to join the filename itself.

If you want a short version of the name for a file, use os.path.basename to extract it.